packages/shared/src/element.ts

```diff
@@ -171,6 +171,9 @@ const VoidTagSet = new Set([
   'wbr'
 ])
 
+const BuiltInTagSet = new Set(['slot', 'component'])
```

Comment:
This should not be placed here since `element.ts` is actually DOM-specific. Just put in `component.ts`.

Reply:
Okay, previously I thought `element.ts` is generalized (including DOM-specific element, component, slot and template).